packages/mds-types/index.ts

```diff
@@ -46,28 +46,10 @@ export const VEHICLE_STATES = [
   'reserved',
   'unknown'
 ] as const
-export type VEHICLE_STATE = typeof VEHICLE_STATES[number]
-
-// export const RIGHT_OF_WAY_STATUSES = ['available', 'reserved', 'unavailable', 'trip']
-export const RIGHT_OF_WAY_STATES = ['available', 'reserved', 'non_operational', 'trip'] as const
+export type VEHICLE_STATE = typeof VEHICLE_STATES_1_0_0[number]
+export const VEHICLE_STATES = VEHICLE_STATES_1_0_0
```

Comment:
May be worth adding a comment (in Typedoc style!) saying that these should be aliases to the latest version supported. I could imagine someone forgetting to update the alias when doing an upgrade at some point.